### PR TITLE
Start of forest animations

### PR DESCRIPTION
**Pull request by Mholliday6611**

### CODE REVIEW

File: Chase.gd [hunk 1]
```diff
@@ -0,0 +1,15 @@
+extends Node2D
+
+@onready var Player
```
Comment (davis-trem) [inline Chase.gd]:
```suggestion
@onready var player_animation = $Player/PlayerAnimation
```

File: Chase.gd [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	Player = self.get_node("Player")
+	Player.get_node('PlayerAnimation').play("run")
```
Comment (davis-trem) [inline Chase.gd]:
```suggestion
	player_animation.play("run")
```

File: Chase.gd [hunk 3]
```diff
@@ -0,0 +1,15 @@
+extends Node2D
```
Comment (davis-trem) [inline Chase.gd]: Move file to `/scripts`

File: scripts/qte_manager.gd [hunk 4]
```diff
@@ -100,6 +100,8 @@ func _start_mashing_event():
 	($TargetRect as ColorRect).position.x = target_mashing_pos
 	
 	$VBoxContainer/ProgressBar.value = 0
+	var scene = preload("res://scenes/chase.tscn").instantiate()
```
Comment (davis-trem) [inline scripts/qte_manager.gd]: Manually add this scene to `qte_mashing.tscn`

**Review by davis-trem — CHANGES_REQUESTED** [review]

File: Job.tscn [hunk 5]
```diff
@@ -0,0 +1,14 @@
+[gd_scene load_steps=3 format=3 uid="uid://bii6mxi26g43c"]
```
Comment (davis-trem) [inline Job.tscn]: Is this file being used?

File: job_posting.tscn [hunk 6]
```diff
@@ -0,0 +1,33 @@
+[gd_scene load_steps=2 format=3 uid="uid://dbd8xrvyd5bib"]
```
Comment (davis-trem) [inline job_posting.tscn]: Move file to `/scenes`

File: scenes/Job.tscn [hunk 7]
```diff
@@ -1,12 +1,6 @@
-[gd_scene load_steps=3 format=3 uid="uid://c7yyhgci03vo0"]
+[gd_scene load_steps=2 format=3 uid="uid://c7yyhgci03vo0"]
```
Comment (davis-trem) [inline scenes/Job.tscn]: Is this scene being used?

File: scenes/bear.tscn [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+@onready var bear_animation = $Bear/AnimationPlayer
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	bear_animation.play(\"chase\")
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	pass
```
Comment (davis-trem) [inline scenes/bear.tscn]: Did you add this code manually?

Comment (Mholliday6611) [reply scenes/bear.tscn]: Yeah, should the play be done another way?

File: Job.tscn (hunk 5 above)
Comment (Mholliday6611) [reply Job.tscn]: Might've stayed during the merge/rebase, I'll delete it

File: scenes/bear.tscn (hunk 8 above)
Comment (Mholliday6611) [reply scenes/bear.tscn]: lol I actually have zero clue

Comment (Mholliday6611) [reply scenes/bear.tscn]: I'm guessing maybe I have to remove the script first in the inspector then delete the file? Cuz I had a bear.gd for a while then I realized I was doing the player animation in the other scene so I delete the script file, then went in the inspector and clicked clear but just now when I looked back there was this in the inspector attached the Bear node. but honestly idk really lol
image

File: scenes/Job.tscn [hunk 9]
```diff
@@ -1,12 +1,34 @@
...
+
+#TEMP FUNCTION TO TEST PROCEEDING ROUNDS
+func _on_texture_rect_gui_input(event):
+	if  event is InputEventMouseButton and event.pressed and event.button_index == MOUSE_BUTTON_LEFT:
+		GameState.player_stats.money += GameState.job_list[job_idx][\"job_payout\"]
+		print(GameState.player_stats.round)
+		GameState.proceedToNextRound()
+		print(GameState.player_stats.round)
+		var scene = preload(\"res://scenes/bedroom.tscn\").instantiate()
+		GameState.scene_swapper(scene)
```
Comment (davis-trem) [inline scenes/Job.tscn]: Why does code keep ending up in your tscn files?

Comment (davis-trem) [reply scenes/Job.tscn]: Ohh I know. It's because you deleted `Job.gd`. Delete this scene too

Comment (Mholliday6611) [reply scenes/Job.tscn]: lmao I  definitely deleted that file, idk Imma close all my tabs

**Pull request merged**